python/cusignal/estimation/filters.py

```diff
@@ -200,8 +199,14 @@ def __init__(
     ):
 
         # Check CuPy version
-        ver = pkg_resources.get_distribution("cupy").version
-        if ver != "8.0.0b4" and ver != "8.0.0rc1" and ver != "8.0.0":
+        # Update to only check for v8.X in cuSignal 0.16
+        ver = cp.__version__
+        if (
+            ver != "8.0.0b4"
+            and ver != "8.0.0b5"
+            and ver != "8.0.0rc1"
+            and ver != "8.0.0"
+        ):
```

Comment:
Is it easier/better to define a set of valid versions up front and then do a
if ver in good_set: ....

Reply:
Smart!